urllib3/fields.py

```diff
@@ -130,7 +130,7 @@ def _render_parts(self, header_parts):
             iterable = header_parts.items()
 
         for name, value in iterable:
-            if value:
+            if value or value == '':
```

Comment:
I think it's more accurate if it's not a falsy value in general (other than empty string), since dealing with an empty list, set, etc. doesn't really make sense.